insurance.go

```diff
@@ -82,6 +79,37 @@ func (c *Client) ListInsurancesWithContext(ctx context.Context, opts *ListOption
 	return
 }
 
+// GetNextInsurancePage returns the next page of addresses
```

Comment:
```suggestion
// GetNextInsurancePage returns the next page of insurance.
```

Please double-check all the docstrings in every file to make sure they are correct after copy-paste.

Reply:
Good catch, thanks!